### PR TITLE
Apply PT-BR translations

### PR DESCRIPTION
**Pull request by tupaschoal**
Applying PT-BR changes too all open issues that hadn't been applied yet:
Applying for issue(s):
- #1670   
- #1675 
- #1679 
- #1691 
- #1697 
- #1698 
- #1704 
- #1713 
- #1718 
- #1721 
- #1725 
- #1737

### CODE REVIEW

File: data/language/pt-BR.txt [hunk 1]
```diff
@@ -723,10 +723,10 @@ STR_1339    :{BLACK}Nenhum resultado de teste ainda...
 STR_1340    :{WINDOW_COLOUR_2}Velocidade máx.: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Duração da atração: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} /
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: This string needs a trailing space. Same for 1346.

File: data/language/pt-BR.txt [hunk 2]
```diff
@@ -1108,8 +1108,8 @@ STR_1726    :Região não está à venda!
 STR_1727    :Direitos de construção não está à venda!
 STR_1728    :Não pode comprar os direitos de construção aqui...
 STR_1729    :Os terrenos não são propriedade do parque!
-STR_1730    :{RED}Fechado - - 
-STR_1731    :{WHITE}{STRINGID} - - 
+STR_1730    :{RED}Fechado
+STR_1731    :{WHITE}{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Needs trailing space, same for 1778.

File: data/language/pt-BR.txt [hunk 3]
```diff
@@ -2106,8 +2106,8 @@ STR_2779    :Janela de Exibição #{COMMA16}
 STR_2780    :Janela de Exibição Grande
 # End of new strings
 STR_2781    :{STRINGID}:{MOVE_X}{195}{STRINGID}
-STR_2782    :SHIFT + 
-STR_2783    :CTRL + 
+STR_2782    :SHIFT +
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Needs trailing space, and so do all others (SHIFT, ALT, CMD)

File: data/language/pt-BR.txt [hunk 4]
```diff
@@ -2386,18 +2386,18 @@ STR_3143    :{SMALLFONT}{BLACK}Mostrar pessoas no mapa
 STR_3144    :{SMALLFONT}{BLACK}Mostrar atrações e barracas no mapa
 STR_3160    :{SMALLFONT}{BLACK}Seleciona o número de circuitos por passeio
 STR_3162    :Impossível alocar memória o suficiente
-STR_3163    :Instalando novos dados: 
+STR_3163    :Instalando novos dados:
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: Needs trailing space. 3169 too.

**Comment by Gymnasiast** [comment]
Looks good, but you removed some trailing space that was supposed to be there. If you address the line notes, I'll merge it.

**Comment by tupaschoal** [comment]
Argh, Atom removed them automatically, I'll backout the trailing space changeset and it should be fine

**Comment by tupaschoal** [comment]
Done, I've reverted back all necessary whitespaces

**Comment by Gymnasiast** [comment]
Ok, that's the PR merged. The repository will be automatically synced to OpenRCT2/OpenRCT2 in about 6.5 hours, after which your changes will be visible in the game itself.

**Pull request merged**